Pod/Classes/PINRemoteImageManager.h

```diff
@@ -108,31 +108,31 @@ typedef void(^PINRemoteImageManagerAuthenticationChallengeCompletionHandler)(NSU
  @param challenge An object that contains the request for authentication.
  @param aHandler A PINRemoteImageManagerAuthenticationChallengeCompletionHandler, see example for further details.
  */
-typedef void(^PINRemoteImageManagerAuthenticationChallenge)(NSURLSessionTask *task, NSURLAuthenticationChallenge *challenge, PINRemoteImageManagerAuthenticationChallengeCompletionHandler aHandler);
+typedef void(^PINRemoteImageManagerAuthenticationChallenge)(NSURLSessionTask * __nonnull task, NSURLAuthenticationChallenge * __nonnull challenge, PINRemoteImageManagerAuthenticationChallengeCompletionHandler __nullable aHandler);
 
 @interface PINRemoteImageManager : NSObject
 
-@property (nonatomic, readonly) PINCache *cache;
+@property (nonatomic, readonly) PINCache * __nonnull cache;
```

Comment:
Is the PINCache really `nullable`? The standard implementation is relying on PINCache heavily